ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -538,6 +540,8 @@ private void assertSuccess(ReadableStreamChannel readableStreamChannel, CountDow
       ByteBufferAsyncWritableChannel asyncWritableChannel = new ByteBufferAsyncWritableChannel();
       long written;
       Future<Long> readIntoFuture = readableStreamChannel.readInto(asyncWritableChannel, null);
+      Assert.assertTrue("ReadyForPollCallback should have been invoked as readInto() was called",
```

Comment:
The test just verifies that the callback is invoked. The `MockNetworkClient` is just a helper class for that.